feature-libs/quote/components/details/overview/quote-details-overview.component.html

```diff
@@ -1,78 +1,69 @@
-  <div class="cx-quote-details-header-status">
-    <h3 class="quote-details-header">
+  <div class="cx-header-container">
+    <h3 class="cx-header">
       {{ 'quote.commons.id' | cxTranslate }}:
       {{ quoteDetails.code }}
     </h3>
-    <h3 class="status">
-      {{ 'quote.commons.status' | cxTranslate }}
+    <h3 class="cx-status">
+      {{ 'quote.commons.status' | cxTranslate }}:
       {{ 'quote.states.' + quoteDetails.state | cxTranslate }}
     </h3>
   </div>
-  <div class="cx-quote-overview">
-    <div class="container">
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(quoteDetails.code, 'quote.details.code') | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.creationTime | cxDate,
-              'quote.details.created'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.updatedTime | cxDate,
-              'quote.details.lastUpdated'
-            ) | async
-          "
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              getTotalPrice(quoteDetails),
-              getTotalPriceDescription(quoteDetails)
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          class="cx-card-description"
-          [content]="
-            getCardContent(
-              quoteDetails.description,
-              'quote.details.description'
-            ) | async
-          "
-          [truncateText]="true"
-          [charactersLimit]="30"
-        ></cx-card>
-      </div>
-      <div class="cx-summary-card">
-        <cx-card
-          [content]="
-            getCardContent(
-              'quote.states.' + quoteDetails.state | cxTranslate,
-              'quote.commons.status'
-            ) | async
-          "
-        ></cx-card>
-        <cx-card
-          [content]="
-            getCardContent(
-              quoteDetails.expirationTime | cxDate,
-              'quote.details.expiryDate'
-            ) | async
-          "
-        ></cx-card>
-      </div>
+  <div class="cx-container">
+    <div class="cx-summary-card">
+      <ng-container *ngIf="!editMode">
+        <div class="cx-content">
+          <cx-card
+            [content]="
+              getQuoteInformation(quoteDetails.name, quoteDetails.description)
+                | async
+            "
+          ></cx-card>
+        </div>
+        <div
+          *ngIf="isQuoteInformationEditable(quoteDetails)"
+          class="cx-edit-step"
+        >
+          <button
+            class="btn btn-link cx-action-link"
+            (click)="toggleEditMode()"
+          >
+            <cx-icon [type]="iconTypes.PENCIL"></cx-icon>
+          </button>
+        </div>
+      </ng-container>
```

Comment:
you could can save on hierarchy level by merging ng-container and div like below
```suggestion
        <div class="cx-content" *ngIf="!editMode">
          <cx-card
            [content]="
              getQuoteInformation(quoteDetails.name, quoteDetails.description)
                | async
            "
          ></cx-card>
        </div>
        <div
          *ngIf="!editMode && isQuoteInformationEditable(quoteDetails)"
          class="cx-edit-step"
        >
          <button
            class="btn btn-link cx-action-link"
            (click)="toggleEditMode()"
          >
            <cx-icon [type]="iconTypes.PENCIL"></cx-icon>
          </button>
        </div>
```

Reply:
@Uli-Tiger, I would prefer to stick to the current solution.
According to my point of you it makes the code more readable.